### ISSUE TITLE
Chant List page: some links open in new tabs, others don't

### ISSUE THREAD

**Issue by jacobdgm** [DDMAL/CantusDB#613]
Tiny issue: on the Chant List page, the Genre and Office links open in new windows, while the Feast link opens in the same window. There's no reason that these links should behave differently.

**Comment by jacobdgm** [comment]
assigning a label of "enhancement" - there's nothing functionally wrong here, but we want our website to behave predictably.

**Comment by dchiller** [comment]
This no longer seems to be the case on the Chant Search, Source Browse Chants, and the Source Search Chants page. Links in these columns navigate to the page in the open tab.
